### ISSUE TITLE
"Failed to contact storage server" message doesn't appear on HTTPS sites?

### ISSUE THREAD

**Issue by gregkare** [remotestorage/remotestorage.js#870]
## Steps to reproduce:
- Go to https://sharesome.5apps.com/ (or https://remotestorage-browser.5apps.com/, or https://grouptabs.5apps.com/)
- Try to connect to remoteStorage using test@gmail.com
- No error message appears in the remoteStorage widget
## Expected result

Trying the same thing on http://litewrite.net gives the following error message in the remoteStorage widget:

> Failed to contact storage server.

It looks like the error message doesn't appear on https websites, but I'm not sure it's really the cause.

**Comment by skddc** [comment]
It should actually say that it couldn't find storage info on that server. Only a timeout should show the message you posted.

**Comment by gregkare** [comment]
The console says it couldn't find info on that server. So maybe it's two bugs?

**Comment by xMartin** [comment]
```
GET https://5apps.com/.well-known/webfinger?resource=acct:doesnotexist@5apps.com 404 (Not Found)

GET https://5apps.com/.well-known/host-meta?resource=acct:doesnotexist@5apps.com 404 (Not Found)

GET https://5apps.com/.well-known/host-meta.json?resource=acct:doesnotexist@5apps.com 404 (Not Found)

Mixed Content: The page at 'https://grouptabs.5apps.com/#' was loaded over HTTPS, but requested an insecure XMLHttpRequest endpoint 'http://5apps.com/.well-known/webfinger?resource=acct:doesnotexist@5apps.com'. This request has been blocked; the content must be served over HTTPS

Discovery failed R…e.DiscoveryError {message: "No storage information found at that user address."} "No storage information found at that user address."
```

Widget goes back to "initial" state.

**Comment by skddc** [comment]
This has been fixed in [0.14.0](https://github.com/remotestorage/remotestorage.js/releases/tag/0.14.0), but we forgot to close the issue.
